tests/first_test.spec.ts

```diff
@@ -21,13 +21,13 @@ test('​Verify user can view product details', async ({ page }) => {
 
   await page.goto(''); 
 
-  await homePage.clickOnItemCardByName("Combination Pliers");
+  await homePage.clickOnItemCardByName('Combination Pliers');
 
   await expect(page).toHaveURL(/.*\/product\/.*/);
-  await expect(page.getByTestId("product-name")).toHaveText("Combination Pliers"); 
-  await expect(page.getByTestId("unit-price")).toHaveText('14.15');
+  await expect(page.getByTestId('product-name')).toHaveText('Combination Pliers'); 
```

Comment:
Let's create ProductDetails page object and move this to it

```
page.getByTestId('product-name')
```

The same below